### ISSUE TITLE
Exporting e-mailaccount results in files of 0 bytes

### ISSUE THREAD

**Issue by nvanderperren** [thundernest/import-export-tools-ng#49]
Hi, I'm trying to export all the e-mails of a Gmail account in Thunderbird. When I choose the option 'Export all folders' it results in a MBOX file per folder. These MBOX-files are 0 bytes and contain no messages, although there are messages in the folders. (screenshots)

![Schermafbeelding 2019-12-09 om 11 24 52](https://user-images.githubusercontent.com/17183054/70428634-36bce180-1a77-11ea-9e2c-2b1f491f20fd.png)
![Schermafbeelding 2019-12-09 om 11 26 37](https://user-images.githubusercontent.com/17183054/70428638-37557800-1a77-11ea-9139-db0289eca82d.png)

I don't encouter this issue when _exporting the files of all folders with structure_, but it looks like the emails of subfolders are not included.

![Schermafbeelding 2019-12-09 om 11 27 56](https://user-images.githubusercontent.com/17183054/70428650-41777680-1a77-11ea-88c7-4469596a35a6.png)
![Schermafbeelding 2019-12-09 om 11 28 24](https://user-images.githubusercontent.com/17183054/70428651-42100d00-1a77-11ea-93f0-49d9f391bdc0.png)

**Comment by cleidigh** [comment]
@nvanderperren 
Sorry for the slow response!
This appears to be slightly different than some similar issues.  Are you able to export a single correctly?  Also it looks like you are on OS X?

**Comment by nvanderperren** [comment]
yes, I'm using OS X (macOS High Sierra). Oddly enough, it's working again. All folders are exported and contain e-mails. 14 days ago, the folder 'Alle email' was 0 bytes, now it's 1.5 MB (screenshot).

<img width="407" alt="Schermafbeelding 2019-12-23 om 14 58 25" src="https://user-images.githubusercontent.com/17183054/71361913-b7f69700-2594-11ea-8eae-9466f714e669.png">
